package.json

```diff
@@ -31,6 +31,7 @@
     "prettier": "prettier *.md *.json \"{docs,syntaxes,.github,.vscode}/**/*.{md,yml,json}\" \"test/**/*.{ts,json}\"",
     "eslint": "eslint ./test/**/*.ts",
     "clean": "rimraf ./dist ./out && npm -w client run clean && npm -w server run clean",
+    "prepackage": "ts-node scripts/interpolateReadmes.ts",
```

Comment:
This makes sense for the vscode extension release, but won't be run before the language server npm package release or the coc-solidity npm package release.

Maybe we add a `generate-readmes` task, make `prepackage` call it (safe but not necessary) and add a line to the `publish-extension.md` to invoke `generate-readmes` before bundling.

Or maybe we just need a to add `publish:coc-solidity` style commands at root and hang the readme generation off that?